### ISSUE TITLE
Alt + F10 with open dropdown results in the error in the console

### ISSUE THREAD

**Issue by dufipl** [ckeditor/ckeditor5#12339]
## 📝 Provide detailed reproduction steps (if any)

1. Open http://127.0.0.1:8080/ckeditor5/latest/examples/builds/classic-editor.html
2. Click on the heading dropdown / the table dropdown / the media embed dropdown etc.
3. Press `Alt` + `F10`

### ✔️ Expected result

Ability to access dropdown by keyboard

### ❌ Actual result

Error in the console.
![dropdown alt f10](https://user-images.githubusercontent.com/32898551/186617012-a8793a44-3a1d-4960-ac6f-8bd729c410d5.gif)

Chrome:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at snippet.js:4:322083
    at _listener.listenTo.priority (snippet.js:4:132598)
    at Function.fire (snippet.js:4:3066)
    at pr.<anonymous> (snippet.js:4:132466)
    at pr.fire (snippet.js:4:3066)
    at HTMLDivElement.t (snippet.js:4:117600)
```
Firefox:
```
Uncaught CKEditorError: i is undefined
Read more: https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-i is undefined
    _initFocusTracking http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    set http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    fire http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    listenTo http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    fire http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    t http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    attach http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    _addEventListener http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    listenTo http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    listenTo http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    listenTo http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    listenTo http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    listenTo http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    addToolbar http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    _initToolbar http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    init http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    create http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    promise callback*create/< http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    create http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js:4
    <anonymous> http://127.0.0.1:8080/ckeditor5/latest/snippets/examples/classic-editor/snippet.js:4
    <anonymous> http://127.0.0.1:8080/ckeditor5/latest/snippets/examples/classic-editor/snippet.js:4
[snippet.js:4:1072](http://127.0.0.1:8080/ckeditor5/latest/snippets/build-classic-source/snippet.js)
```

Safari: 
```
CKEditorError: undefined is not an object (evaluating 'i.length')
Read more: https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-undefined is not an object (evaluating 'i.length')
```

## 📃 Other details

* Browser: Any
* OS: Mac OS Monterey

**It is a regression.** 
---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by dufipl** [comment]
## Alternative scenario 

1. Open http://127.0.0.1:8080/ckeditor5/latest/examples/builds/balloon-block-editor.html
2. Click on the block balloon dropdown
3. Press `Alt` + `F10`

## Result

```
Uncaught TypeError: Cannot read properties of undefined (reading 'length')
    at snippet.js:4:321764
    at _listener.listenTo.priority (snippet.js:4:132279)
    at Function.fire (snippet.js:4:3066)
    at pr.<anonymous> (snippet.js:4:132147)
    at pr.fire (snippet.js:4:3066)
    at HTMLDivElement.t (snippet.js:4:117600)
```

**Comment by Dumluregn** [comment]
Closed in ckeditor/ckeditor5#12343.
